pkg/kubelet/kubelet_pods_test.go

```diff
@@ -185,17 +185,17 @@ fe00::0	ip6-localnet
 fe00::0	ip6-mcastprefix
 fe00::1	ip6-allnodes
 fe00::2	ip6-allrouters
-123.45.67.89	some.domain
+192.0.2.89	some.domain
 
 # Entries added by HostAliases.
-123.45.67.89	foo	bar	baz
+192.0.2.89	foo	bar	baz
 `,
 		},
 		{
 			"hosts_test_file2_with_host_aliases",
 			[]v1.HostAlias{
-				{IP: "123.45.67.89", Hostnames: []string{"foo", "bar", "baz"}},
-				{IP: "456.78.90.123", Hostnames: []string{"park", "doo", "boo"}},
```

Comment:
just curious, don't we validate ips? this IP is invalid

Reply:
yeah I noticed that too, apparently not